### DISCUSSION TITLE
Block iCloud notification when launching Freeform App

### DISCUSSION THREAD

**Discussion by jmahlman** [usnistgov/macos_security#215] [Ideas] [answered]
At this time there doesn't seem to be a key specifically for disabling iCloud for the Freeform app; however, a suggestion was made on Slack that allows you to block the iCloud prompt when launching Freeform. 

Preference domain: `com.apple.freeform`
Key:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>CRLConnectICloudDialogShown</key>
	<integer>13001000</integer>
</dict>
</plist>
```

Should we include this as part of blocking iCloud? As a note, I am testing this but unable to get it working...but I figure it's a good discussion point.

**Comment by jmahlman** [comment]
Seems the integer may be related to the OS version. The int `13001000` didn't work on my 13.2 test machines but `13002000` does. May not be the best way to do this but let's bring it up anyway.

**Comment by robertgendler** [comment]
After that initial prompt, Im also not seeing a way to sign in within Freeform.

**Comment by jmahlman** [comment, marked as answer]
Setting the integert to `99999999` seems to work:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>CRLConnectICloudDialogShown</key>
	<integer>99999999</integer>
</dict>
</plist>
````

While this doesn't disable the ability to sync to iCloud (using System Settings) it disables the pop-up.
